docs/ru/sql-reference/functions/other-functions.md

```diff
@@ -2088,3 +2088,52 @@ SELECT tcpPort();
 
 -   [tcp_port](../../operations/server-configuration-parameters/settings.md#server_configuration_parameters-tcp_port)
 
+## currentProfiles {#current-profiles}
+
+Возвращает список [профилей настроек](../../operations/access-rights.md#settings-profiles-management) для текущего пользователя. Для изменения текущего профиля настроек может быть использована команда [SET PROFILE](../../sql-reference/statements/set.md#query-set). Если команда `SET PROFILE` не использовалась, функция возвращает профили, указанные при объявлении текущего пользователя (см. [CREATE USER](../../sql-reference/statements/create/user.md#create-user-statement)).
+
+**Синтаксис**
+
+``` sql
+currentProfiles()
+```
+
+**Возвращаемое значение**
+
+-   Список профилей настроек для текущего пользователя. 
+
+Тип: [Array](../../sql-reference/data-types/array.md)([String](../../sql-reference/data-types/string.md)).
+
+## enabledProfiles {#enabled-profiles}
+
+Возвращает профили настроек, назначенные пользователю как явно, так и неявно. Явно назначенные профили — это те же профили, которые возвращает функция [currentProfiles](#current-profiles). Неявно назначенные профили включают родительские профили других назначенных профилей, профили, назначенные с помощью предоставленных ролей, профили, назначенные с помощью собственных настроек, и основной профиль по умолчанию (см. секцию `default_profile` в основном конфигурационном файле).
+
+ Если какой-либо профиль встречается несколько раз (с некоторыми другими настройками между ними), последнее событие перезаписывает все предыдущие.
+
+**Синтаксис**
+
+``` sql
+enabledProfiles()
+```
+
+**Возвращаемое значение**
+
+-   Список доступных профилей для текущего пользователя. 
+
+Тип: [Array](../../sql-reference/data-types/array.md)([String](../../sql-reference/data-types/string.md)).
+
+## defaultProfiles {#default-profiles}
+
+Возвращает все профили, указанные при объявлении текущего пользователя (см. [CREATE USER](../../sql-reference/statements/create/user.md#create-user-statement), секция `SETTINGS`)
```

Comment:
```suggestion
Возвращает все профили, указанные при объявлении текущего пользователя (см. [CREATE USER](../../sql-reference/statements/create/user.md#create-user-statement), секция `PROFILE`)
```

Reply:
Not sure. It's `SETTINGS` section, `PROFILE` is a parameter.

Reply:
I meant `CREATE USER ... SETTINGS PROFILE profile_name`.
The documentation doesn't have this section, so I think a link to `CREATE USER` is enough here.